server/middleware.js

```diff
@@ -0,0 +1,17 @@
+const middleware = {
```

Comment:
Can we rename to something more meaningful? `checkClientVersionMiddleware` or something?